src/CommandLine.cpp

```diff
@@ -36,56 +71,70 @@ namespace tsm::cli {
 
         return std::make_optional(result);
     }
-    catch (const cxxopts::OptionException &e)
-    {
+    catch (const cxxopts::OptionException& e) {
         std::cerr << e.what() << std::endl;
         return std::nullopt;
     }
-    catch (...)
-    {
-        std::cerr << "Unhandled execption with parsing command-line options." << std::endl;
+    catch (...) {
+        std::cerr << "Unhandled execption when parsing command-line options." << std::endl;
         return std::nullopt;
     }
 }
 
 /***********************************************************************************/
-std::string cleanRegexPattern(const std::string& inputPattern) {
-    std::string cleaned = inputPattern;
-    cleaned.erase(std::remove(cleaned.begin(), cleaned.end(), '\''), cleaned.end());
-
-    return cleaned;
-}
-
-/***********************************************************************************/
-bool CLIOptions::verify() const {
-
-    if (DatasetName.empty()) {
+bool CommandLine::verify(const cxxopts::ParseResult& result, const std::vector<fs::path> STDINfileList) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'STDINfileList' should be passed by const reference.](https://app.codacy.com/manual/htmlboss/netcdf-timestamp-mapper/pullRequest?prid=5068050)